tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -1118,6 +1118,7 @@ bool TangoRosNode::GetMapName(
     const tango_ros_messages::GetMapName::Request &req,
     tango_ros_messages::GetMapName::Response &res) {
   res.map_name = GetMapNameFromUuid(req.map_uuid);
+  return res.map_name.empty();
```

Comment:
Is this return value used anywhere? From the function name `GetMapName`, I would expect that it returns the map name instead of a boolean. Suggestion: rename to `GetMapNameServiceRequestCallback`.